recipes-bsp/acpi-tables/files/edison/acpi-tables-load

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# This script copies all ACPI tables into /kernel/firmware/acpi
+# You must have configfs enabled for this to work
+
+    for TABLE in `find /kernel/firmware/acpi/ -type f -name "*.aml" 2>/dev/null`; do
```

Comment:
I would rather prefer to see something like
`find /kernel/firmware/acpi/ -type f -name "*.aml" 2>/dev/null | while read TABLE; do`

    

> ``

Reply:
Ok

Reply:
I missed this one. Fixed it now.